clients/cloud/scala/README.md

```diff
@@ -0,0 +1,112 @@
+# Scala Producer and Consumer for Confluent Cloud
+
+This directory includes projects demonstrating how to use the Scala producer and consumer with Confluent Cloud.
+
+For more information, please see the [application development documentation](https://docs.confluent.io/current/api-javadoc.html)
+
+
+# Quickstart
+
+Before running the examples, **you should setup the Confluent Cloud CLI** by running the [quickstart](https://docs.confluent.io/current/quickstart/cloud-quickstart.html#cloud-quickstart).
+
+
+
+
+1. Now we can turn our attention to the client examples in this directory.
+
+	1. First run the example consumer
+		```shell
+		$ cd examples/clients/cloud/scala
+		# Build the client examples
+		$ sbt clean compile
+		
+		# Run the consumer
+		$ sbt 'runMain io.confluent.examples.clients.scala.Consumer /path/to/.ccloud/config testtopic'
```

Comment:
Should we use `~/.ccloud/config` here or at least hint the user that is the config file they probably want to use?

Reply:
You could use:
`$ sbt "runMain io.confluent.examples.clients.scala.Consumer $HOME/.ccloud/config testtopic"`

Reply:
Done, thanks